data/date.go

```diff
@@ -0,0 +1,64 @@
+package data
+
+import (
+	"strconv"
+	"time"
+)
+
+type Date struct {
+	date       time.Time
+	ys, ms, ds string
+}
+
+const DateFormat = "2006-01-02"
+
+func MustParseDate(dateAsString string) Date {
```

Comment:
Public functions should have a godoc

Reply:
Not sure how the godoc looks as I can't find any of the other public method's godocs. Where can I find examples?

Reply:
It's a comment that the IDE interprets and tells you what the function does
```suggestion
// MustParseDate is a function that...
func MustParseDate(dateAsString string) Date {
```